src/ring_theory/polynomial/cyclotomic/basic.lean

```diff
@@ -857,7 +858,7 @@ lemma cyclotomic_eq_minpoly_rat {n : ℕ} {K : Type*} [field K] {μ : K}
   cyclotomic n ℚ = minpoly ℚ μ :=
 begin
   rw [← map_cyclotomic_int, cyclotomic_eq_minpoly h hpos],
-  exact (minpoly.gcd_domain_eq_field_fractions _ (is_integral h hpos)).symm
+  simpa using (minpoly.gcd_domain_eq_field_fractions' ℚ (is_integral h hpos)).symm
```

Comment:
I thought `minpoly.gcd_domain_eq_field_fractions'` would be the same as the old `minpoly.gcd_domain_eq_field_fractions`, why do we need a `simpa` here?

Reply:
Because I did this before writing `minpoly.gcd_domain_eq_field_fractions'` and I then forgot to fix it, thanks.